pom.xml

```diff
@@ -228,6 +228,13 @@
             <artifactId>velocity-tools</artifactId>
             <version>1.4</version>
         </dependency>
+
+        <dependency>
+            <groupId>oro</groupId>
+            <artifactId>oro</artifactId>
+            <version>2.0.8</version>
+        </dependency>
```

Comment:
You may need to do a `mvn clean install` first or if you're using an IDE reload or refresh the project. Looks like it is not downloading the dependency.